src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2814,10 +2820,6 @@ private void EnsureDescriptorsInitialized()
                 // get the metadata via reflection.
                 Debug.Assert(m_rawManifest == null);
 #if !ES_BUILD_STANDALONE
-                [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

Comment:
Is this going to be long term? If so, can we remove the local method that was introduced just so we could suppress the warning here?

cc @LakshanF